syntax = "proto3";

package demo;

service DemoService {
  rpc DemoAPI(DemoRequest) returns (DemoResponse);
}

message DemoRequest { string query = 1; }

message DemoResponse { string message = 1; }